app/src/main/res/xml/keys_letters_turkish.xml

```diff
@@ -68,11 +68,11 @@
             app:topSmallNumber="6" />
         <Key
             app:keyLabel="u"
-            app:popupCharacters="û7ü"
+            app:popupCharacters="ûü7"
             app:popupKeyboard="@xml/keyboard_popup_template"
             app:topSmallNumber="7" />
         <Key
-            app:keyLabel="i"
+            app:keyLabel="iİ"
```

Comment:
You have added it to the key label, not popup characters.

> - [x] I manually tested my changes on device/emulator (if applicable).

This is in the template for a reason. Please test your changes before you commit and push.